Google.Ads.GoogleAds.Extensions/src/DependencyInjection/GoogleAdsClientServiceCollectionExtensions.cs

```diff
@@ -6,6 +7,9 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
+    /// <summary>
```

Comment:
nit: same cleanup here and in GoogleAdsConfig.cs

Reply:
Done.